src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -68,6 +69,9 @@ class JenkinsJobManager {
             if (startOnCreate && missingJob.jobName.contains("Rev.com-featurebranch-build-feature")) {
```

Comment:
yeah sounds better to use the extra effort later to do that. Also at first glance that looks awesome. Hopefully it will work out for us.